src/pages/academy/grading/subcomponents/GradingSubmissionFilters.tsx

```diff
@@ -0,0 +1,45 @@
+import { Icon } from '@blueprintjs/core';
+import { IconNames } from '@blueprintjs/icons';
+import { ColumnFilter, ColumnFiltersState } from '@tanstack/react-table';
+import { Badge, Flex } from '@tremor/react';
+
+import { badgeColor } from './colors';
+
+type GradingSubmissinoFiltersProps = {
+  filters: ColumnFiltersState;
+  onFilterRemove: (filter: ColumnFilter) => void;
+};
+
+const GradingSubmissionFilters: React.FC<GradingSubmissinoFiltersProps> = ({
+  filters,
+  onFilterRemove
+}) => {
+  return (
+    <Flex justifyContent="justify-start" spaceX="space-x-1">
+      {filters.map(filter => (
+        <FilterBadge filter={filter} onRemove={onFilterRemove} key={filter.id} />
+      ))}
+    </Flex>
+  );
+};
+
+type FilterBadgeProps = {
+  filter: ColumnFilter;
+  onRemove: (filter: ColumnFilter) => void;
+};
+
+const FilterBadge: React.FC<FilterBadgeProps> = ({ filter, onRemove }) => {
```

Comment:
do you think it will be better if we put `FilterBadge` together with the other badges in `GradingBadges`?

this will centralize all badges into a single place/file. additionally, we wouldn't need to separate out color handling functions to a separate `colors.ts` file anymore, since it is only being used by badge components.